shell/app-shell/elements/fb-data/fb-user-context.js

```diff
@@ -156,10 +156,19 @@ class FbUserContextElement extends Xen.Debug(Xen.Base, log) {
       if (store) {
         // compute description
         const typeName = store.type.toPrettyString().toLowerCase();
-        store.description = ArcsUtils._getHandleDescription(typeName, store.tags, 'my', ownername);
+        store.description = this._getStoreDescription(typeName, store.tags, 'my', ownername);
       }
     }
   }
+  _getStoreDescription(name, tags, user, owner) {
+    let proper = (user === owner) ? 'my' : `${owner}'s`;
```

Comment:
Was this something where we thought a feature request for runtime to generate this made sense? I remember us talking about it briefly.

Reply:
Yes, this is absolutely something the runtime needs to do.

Reply:
https://github.com/PolymerLabs/arcs/issues/1615